trieste/acquisition/function/__init__.py

```diff
@@ -35,6 +35,7 @@
 )
 from .function import (
     AugmentedExpectedImprovement,
+    BatchExpectedImprovement,
```

Comment:
The other AFs also expose the AF type (`batch_expected_improvement` in this case). Not totally sure why, but let's be consistent.